# ManualParameter.msg 手动控制参数
float64 motorSpeed1      #速度：mm/s
float64 steeringAngle1   #角度：度
float64 forkCommand      #货叉命令